driver/src/contracts/gnosis_safe_gas_station.rs

```diff
@@ -0,0 +1,82 @@
+use ethcontract::U256;
+use serde::Deserialize;
+use std::time::Duration;
+use uint::FromDecStrErr;
+
+/// Result of https://safe-relay.gnosis.io/ api call `/v1/gas-station`.
+///
+/// Prices are in wei.
+#[derive(Deserialize, Debug, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct GasPrice {
+    pub last_update: String,
+    #[serde(deserialize_with = "deserialize_u256_from_string")]
```

Comment:
Can we use [U256::from_dec_str])https://docs.rs/ethereum-types/0.2.3/ethereum_types/struct.U256.html#method.from_dec_str) here?

Reply:
It is used by that function. It cannot be used at this level because this function doesn't have the correct parameter and return types.

Reply:
Oops, yeah I saw that and forgot to remove this comment. Never mind.